src/Model/QuestionManager.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Model;
+
+class QuestionManager extends AbstractManager
+{
+    
+    const TABLE = 'question';
+
+    /**
+     *  Initializes this class.
+     */
+    public function __construct()
+    {
+        parent::__construct(self::TABLE);
+    }
+
+    /**
+     * Selects a random question.
+     * If table is empty or does not exist or if we encounter an error, returns an error message.
+     * @return array
+     */
+    public function selectOneRandom(): array
+    {
+        try {
+            $statement = $this->pdo->query("SELECT * FROM " . self::TABLE . " ORDER BY RAND() LIMIT 1");
+        } catch (\Exception $e) {
+            return ["id" => 0 , "question" => "Je n'ai pas trouvé de question (╥﹏╥) Essaies d'en ajouter une? ^_^"];
+        }
+
+        if ($statement === false) {
+            return ["id" => 0 , "question" => "Je n'ai pas trouvé de question (╥﹏╥) Essaies d'en ajouter une? ^_^"];
```

Comment:
Don't repeat yourself.
If you want a fake question for error handling and repeat it, you probably should have a constant.